app/ui/src/app/store/step/step.store.ts

```diff
@@ -94,10 +93,15 @@ export class StepStore {
       description: "Sends a message to the integration's log",
       configuredProperties: undefined,
       properties: {
-        message: {
+        expression: {
           type: 'string',
           displayName: 'Log Message',
-          required: true,
+          required: false,
+        },
+        bodyLoggingEnabled: {
+          type: 'checkbox',
```

Comment:
gashcrumb: Do you have any idea why this appears as a text box?

Reply:
whoops, yeah do `type` boolean for now, need to make the form service also accept `checkbox` properly :-)